Quantidade de VERIFICAR: 0
Error in `mutate()`:
i In argument: `Check_covas = Stand_tree_ha/(Pits_ha/Pits_por_sob)`.
Caused by error:
! objeto 'Pits_por_sob' n√£o encontrado
Run `rlang::last_trace()` to see where the error occurred.
